src/app/api.mjs

```diff
@@ -201,3 +204,10 @@ export async function login(data) {
   const response = await client.post(url, data);
   return response.data;
 }
+
+export async function signupDetails(data) {
+  const userId = localStorage.getItem(USER_ID);
+  const url = `${SIGNUP_DATAILS_PATH}/${userId}/`;
```

Comment:
API standard is:
<resource_name>/<object_id>/

I would just pass user_id as a query parameter via `?userId=1`